src/include/execution/vm/bytecode_handlers.h

```diff
@@ -34,6 +34,11 @@
 #define VM_OP_WARM VM_OP inline
 #define VM_OP_COLD VM_OP NEVER_INLINE
 
+namespace terrier::execution::vm {
+STRONG_TYPEDEF(col_id_t, uint16_t);
```

Comment:
This is already defined in `include/storage/storage_defs.h`. We don't want another definition in a different namespace.

Reply:
I have fix this.